chpl-src/announcing-chapel-2.7.chpl

```diff
@@ -0,0 +1,288 @@
+// Announcing Chapel 2.7!
+// authors: ["Jade Abraham", "Engin Kayraklioglu", "Daniel Fedorin", "Ben Harshbarger", "Brad Chamberlain"]
+// summary: "Highlights from the December 2025 release of Chapel 2.7"
+// tags: ["Release Announcements", "Debugging", "Tools", "Dyno"]
+// date: 2025-12-18
+/*
+
+  The Chapel developer community is pleased to announce the release of
+  Chapel 2.7!  Notably, this is the first version of Chapel to be made
+  available since our project [joined the High Performance Software
+  Foundation](https://hpsf.io/blog/2025/hpsf-welcomes-chapel/) this
+  fall.
+
+  As always, you can [download and
+  install](https://chapel-lang.org/download/) this new version in a
+  {{<sidenote "right" "variety of formats">}}Please note that some
+  formats may not yet be available at time of
+  publication...{{</sidenote>}}, including Spack, Docker, Homebrew,
+  various Linux package managers, and source tarballs.
+
+  This article introduces some of the highlights of Chapel 2.7,
+  including:
+
+  * A new compiler flag supporting [vector libraries](#vector-library)
+
+  * Improvements when [debugging Chapel programs](#debugging-enhancements)
+
+  * [Stack traces](#stack-traces) on fatal errors by default
+
+  * Enhancements to [Mason](#mason), Chapel's package manager
+
+  * Improvements to the capabilities of the [Dyno compiler
+    front-end](#improvements-to-the-dyno-compiler-front-end)
+
+
+  In addition to the above features, Chapel 2.7 also includes
+improvements to the pre-built [Linux
+packages](https://chapel-lang.org/download/#linux), in terms of
+features and flexibility.
+
+  For more information, or a much more complete list of changes in
+  Chapel 2.7, see the
+  [CHANGES.md](https://github.com/chapel-lang/chapel/blob/release/2.7/CHANGES.md)
+  file.  And as always, thanks to [everyone who
+  contributed](https://github.com/chapel-lang/chapel/blob/release/2.7/CONTRIBUTORS.md)
+  to version 2.7!
+
+
+  ### Using vector libraries
+
+  One of the nice performance-oriented features added in this release
+  improves the vectorization capabilities of the Chapel compiler
+  through vector math libraries.  Chapel already does a good job of
+  {{<sidenote "right" "vectorizing code">}}Vectorization is turning
+  scalar code into vector code to make use modern CPU's SIMD
+  capabilities.{{</sidenote>}}, particularly when making use of vector
+  math libraries such as Libmvec or SVML to accelerate math-heavy
+  code.  However, using such libraries has traditionally not been a
+  very user-friendly experience, requiring users to specify low-level
+  C/LLVM flags to enable this extra performance.
+
+  In this release, we added `--vector-library`, a new Chapel compiler
+  flag that makes it easy to enable vector math libraries in Chapel
+  programs. This flag takes one argument providing the name of the
+  vector library to use. For example, to use Libmvec on x86 systems,
```

Comment:
libmvec should be all lower case

Reply:
Huh. I have never seen it capitalized like that in any other docs before. But given thats the glibc docs, I guess its the source of truth and everything else is wrong. Ok Libmvec it is